### PR TITLE
New feature and style improvements 

### PR DESCRIPTION
**Pull request by babuenir**

### CODE REVIEW

File: sound/soc/bcm/Kconfig [hunk 1]
```diff
@@ -166,6 +166,11 @@ config SND_BCM2708_SOC_ALLO_PIANO_DAC_PLUS
 	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
 	select SND_SOC_PCM512x_I2C
 	help
+          Allo Piano DAC Plus (2.1) driver requires firmware which can
+          be obtained from:
+
+          <https://allo.com/sparky/piano-dac-2-1.html>
```
Comment (pelwell) [inline sound/soc/bcm/Kconfig]: I'm not happy about this URL - the kinds of people who won't know how to install their soundcard (which presumably comes with instructions) will never find this, and it's too much like advertising.

File: sound/soc/bcm/allo-piano-dac-plus.c [hunk 2]
```diff
@@ -463,5 +679,5 @@ static struct platform_driver snd_allo_piano_dac_driver = {
 module_platform_driver(snd_allo_piano_dac_driver);
 
 MODULE_AUTHOR("Baswaraj K <jaikumar@cem-solutions.net>");
-MODULE_DESCRIPTION("ALSA ASoC Machine Driver for Allo Piano DAC Plus");
+MODULE_DESCRIPTION("ASoC Driver for Allo Piano DAC Plus. Device and firmware details in https://allo.com/sparky/piano-dac-2-1.html");
```
Comment (pelwell) [inline sound/soc/bcm/allo-piano-dac-plus.c]: Again - remove the URL.

**Review by pelwell — COMMENTED** [review]
I'm not impressed with the daily updates; you're supposed to submit drivers when they're finished - I don't want to be part of your product development cycle.

**Comment by babuenir** [comment]
@pelwell updated the tree based on your review.

**Comment by pelwell** [comment]
Is this feature-complete, to the best of your knowledge?

File: arch/arm/boot/dts/overlays/README [hunk 3]
```diff
@@ -323,6 +323,9 @@ Params: 24db_digital_gain       Allow gain to be applied via the PCM512x codec
                                 responsibility of the user to ensure that
                                 the Digital volume control is set to a value
                                 that does not result in clipping/distortion!)
+	glb_mclk		This option is only with Kali board. If enabled,
```
Comment (pelwell) [inline arch/arm/boot/dts/overlays/README]: No TABs, please.

**Comment by babuenir** [comment]
@pelwell Yes, it is complete.

**Pull request merged**